src/main/scala/latis/service/hapi/DataService.scala

```diff
@@ -25,33 +26,44 @@ class DataService[F[_]: Concurrent](
   val service: HttpRoutes[F] =
     HttpRoutes.of[F] {
       case GET -> Root / "data"
```

Comment:
More specific error codes (like 1401 instead of 1400) are optional. There's a catch-all case below that returns a 1400. I think it would take extra logic to determine if there was a missing parameter (1400) or an extra, unknown parameter (1401), so I must have opted to not use the more specific error message in this case and just use 1400 for both.